db/migrate/20150806211453_rename_ems_event_table_to_event_stream.rb

```diff
@@ -13,7 +14,13 @@ def up
 
     add_column :event_streams, :type, :string
     say_with_time("Updating Type in EventStreams") do
-      EventStream.update_all(:type => 'EmsEvent')
+      update_count = nil
+      done = 0
+      until update_count == 0
+        update_count = EventStream.where(:type => nil).limit(1000).update_all(:type => 'EmsEvent')
+        done += update_count
+        say "#{done.to_s.reverse.gsub(/...(?=.)/, '\&,').reverse} rows", :subitem if update_count > 0
```

Comment:
Would this create too many console messages?  The test database had about 7 million records, so this would output 7000 messages.

@Fryguy @jrafanie Do we have anything like this in other migrations?